src/main/kotlin/org/malachite/estella/offer/api/OfferController.kt

```diff
@@ -20,8 +20,14 @@ class OfferController(
 
     @CrossOrigin
     @GetMapping
-    fun getOffers(): ResponseEntity<List<OfferResponse>> =
+    fun getOffers(
+        @RequestParam("only_started", required = false) onlyStarted: Boolean
+    ): ResponseEntity<List<OfferResponse>> =
         offerService.getOffers()
+            .let {
+                if (onlyStarted) { it.filter { it.recruitmentProcess?.isStarted() == true } }
+                else it
+            }
```

Comment:
Jakby któryś funkcyjny świr powiedział mi, jak to zrobić lepiej, to byłbym wdzięczny (taki filter ale z dodatkowym warunkiem) XD

Reply:
Po pierwsze wydaje mi się że starczy tak jeśli chodzi o filter
``` kotlin
it.filter { it.recruitmentProcess?.isStarted() }
```
Plus nie wiem o co ci chodzi dokońca żeby to zrobić lepiej bo możesz przenieść dodatkowy warunek do filtra, ale wtedy będzie to trochę mniej wydajne bo w sytuacji gdy chcemy dać całą listę (czyli OnlyStarted = false) po niej ponownie przejdziemy.